### ISSUE TITLE
[Bug] Cannot drive in multiplayer

### ISSUE THREAD

**Issue by zenith391** [DonBruce64/MinecraftTransportSimulator#361]
The latest version of Immersive Vehicles (17.3.0) again ships with a bug that makes cars unable to be driven in multiplayer.  To be more clear we can enter a vehicle and start the engine, however we cannot drive it at all, no gearing, no turning, no throttle, etc. Note that it doesn't causes any crash.
Here's the stack trace i get in the server console whenever i do some driving action (turn, throttle,etc.) if it can help:
```
java.lang.ClassNotFoundException: net.minecraft.client.multiplayer.WorldClient
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101) ~[launchwrapper-1.12.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:419) ~[?:1.8.0_242]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:352) ~[?:1.8.0_242]
        at minecrafttransportsimulator.wrappers.WrapperNetwork$WrapperHandler.onMessage(WrapperNetwork.java:142) ~[WrapperNetwork$WrapperHandler.class:?]
        at minecrafttransportsimulator.wrappers.WrapperNetwork$WrapperHandler.onMessage(WrapperNetwork.java:137) ~[WrapperNetwork$WrapperHandler.class:?]
        at net.minecraftforge.fml.common.network.simpleimpl.SimpleChannelHandlerWrapper.channelRead0(SimpleChannelHandlerWrapper.java:56) ~[SimpleChannelHandlerWrapper.class:?]
        at org.spongepowered.mod.network.brokenmod.BrokenModSimpleNetworkChannelWrapper.lambda$channelRead0$1(BrokenModSimpleNetworkChannelWrapper.java:55) ~[BrokenModSimpleNetworkChannelWrapper.class:1.12.2-2838-7.1.9]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[?:1.8.0_242]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) ~[?:1.8.0_242]
        at net.minecraft.util.Util.func_181617_a(SourceFile:46) ~[h.class:?]
        ... 7 more
```

**Comment by DonBruce64** [comment]
Odd, I can't seem to reproduce this on my end.  Are you on a server?

**Comment by DonBruce64** [comment]
Fixed in V17.3.1.
